### PR TITLE
Adding disable shielded boot flag and disk type ssd flag to enhance the cluster creation

### PR DESCRIPTION
**Pull request by prince-cs**

### CODE REVIEW

**Comment by kuldeepkk-dev** [comment]
Discussed with Prince on disable secure boot part for the tests. Approving since other PRs are waiting on this change.

**Review by kuldeepkk-dev — APPROVED** [review]
LGTM

**Comment by cjac** [comment]
Don't forget rapids and spark-rapids

**Comment by prince-cs** [comment]
> Don't forget rapids and spark-rapids

Okay sure

File: integration_tests/dataproc_test_case.py [hunk 1]
```diff
@@ -118,6 +118,10 @@ def createCluster(self,
             if FLAGS.image_version in self.IMAGE_VERSION_2_2:
                 args.append("--public-ip-address")
 
+        for i in init_actions:
+            if "install_gpu_driver.sh" in i or "mlvm.sh" in i:
```
Comment (cjac) [inline integration_tests/dataproc_test_case.py:122]: rapids.sh, spark-rapids.sh, horovod.sh

**Pull request merged**